mage_ai/cache/block.py

```diff
@@ -150,7 +162,7 @@ def __filter(
                 pipeline_dict = cache_dict.get('pipeline') or {}
 
                 return pipeline_dict.get('uuid') != pipeline_uuid and \
-                    pipeline_dict.get('repo_path') != repo_path
+                    pipeline_dict.get('repo_path') == repo_path
```

Comment:
The `!=` was filtering out all pipelines, resulting in no "shared pipelines" being shown for the blocks.